frontend/src/components/ModelsList.jsx

```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
+import {
+	Button, Typography, FormControl, FormControlLabel, Paper, Checkbox, Input, InputLabel, SnackbarContent, Grid, TableBody, Table, TableRow, TableCell, ExpansionPanelSummary, ExpansionPanel, Divider, ExpansionPanelActions, ExpansionPanelDetails, Chip
+} from '@material-ui/core';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import { Theme } from '@material-ui/core/styles/createMuiTheme';
+import { createStyles, WithStyles } from '@material-ui/core/styles';
+import withStyles from '@material-ui/core/styles/withStyles';
+import classNames from 'classnames';
+import { RouteComponentProps } from 'react-router-dom';
+import { GetModelsResponse, Parameter, Metric, Model as ModelConnect, Model } from "../utils/connect"
+
+const styles = (theme) =>
+	createStyles({
+		left:{
+			textAlign:"left"
+		}
+	});
+
+function ModelsListComp(props) {
+	const { classes } = props;
+	const models = props.models || [];
+	return (
+		<div>
+			{
+				models.map(model =>
+					<ExpansionPanel>
+						<ExpansionPanelSummary expandIcon={<ExpandMoreIcon />}>
+							<Grid container direction="row">
+								<Grid item xs={6}>
+									<Grid container direction="column" spacing={0}>
+										<Grid item xs={12}><Typography className={classes.left}>name: {model.name}</Typography></Grid>
+										<Grid item xs={12}><Typography className={classes.left}>description: {model.description}</Typography></Grid>
+										<Grid item xs={12}><Typography className={classes.left}>author: not implemented</Typography></Grid>
+										<Grid item xs={12}><Typography className={classes.left}>added: not implemented</Typography></Grid>
+									</Grid>
+								</Grid>
+								<Grid item xs={6}>
+									<Grid container direction="column" spacing={0}>
+										<Grid item xs={12}><Typography className={classes.left}>tags: {model.tags.join(", ")}</Typography></Grid>
+										<Grid item xs={12}><Typography className={classes.left}>metrics: {model.metrics.map( m => m.id + "=" + m.value ).join(", ")}</Typography></Grid>
+									</Grid>
+								</Grid>
+							</Grid>
+						</ExpansionPanelSummary>
+						<ExpansionPanelDetails>
+							<Grid container direction="row">
+								<Grid item xs={6}>
+									<Typography className={classes.left}>parameters: {model.parameters.map( m => m.id + "=" + m.value ).join(", ")}</Typography>
+								</Grid>
+								<Grid item xs={6}>
+									<Typography className={classes.left}>hiperparameters: {model.hiperParameters.map( m => m.id + "=" + m.value ).join(", ")}</Typography>
```

Comment:
Please decide on a single, coherent way of describing hyperparameters. Assuming we're releasing the project only in English, one should avoid mixing the Polish "_hiperparametry_" word with correct English spelling, that is "_hyperparameters_".

Reply:
hiperparameters is a hybrid that should work for everyone

Reply:
Well, it does _not_ work for me.